package.json

```diff
@@ -1,28 +1,36 @@
 {
   "private": true,
   "scripts": {
+    "buildDeps": "yarn workspace @openvaa/app-shared build && true",
```

Comment:
Some commands require the app-shared module to be built to run properly.

The && true in the end prevents command line arguments in the pre-commit hook from being passed to this command.